spec/models/claim_spec.rb

```diff
@@ -59,13 +59,13 @@
   end
 
   describe "scopes" do
-    describe "#not_internal" do
+    describe "#visible" do
       it "returns the claims that dont have status internal" do
         create(:claim)
         claim1 = create(:claim, :draft)
         claim2 = create(:claim, :submitted)
```

Comment:
Can you add claims to this test which aren't `:draft` or `:submitted`. As currently it is returning `Claim.all`